r2.3 cherry-pick request: Shift padded NMS compat window forward to fix TFLite conversion

srjoglekar246 opened the pull request:
Since the new implementation does not convert to TFLite, we are extending the compat window so that conversion works in TF2.3.

PiperOrigin-RevId: 319856402
Change-Id: I60eeb4e411609511046cc4676c2f3ad630b3769d

Commit:
Shift forward-compat window for NMS padded to fix TFLite conversion in 2.3

PiperOrigin-RevId: 319856402
Change-Id: I60eeb4e411609511046cc4676c2f3ad630b3769d

mihaimaruseac commented:
Letting @azaks3 decide if the cherry-pick can go in.

Note that there is a Ubuntu CPU failure on the presubmits.

smit-hinsu committed:
Disable MLIR bridge for NMS image ops test

MLIR bridge doesn't support tf.NonMaxSuppressionV4 legalization that is
conditionally generated by non_max_suppression_padded function.

PiperOrigin-RevId: 320197235
Change-Id: If7242133254680b366771ced50de074ed6180563

srjoglekar246 commented:
Added Smit's changes for presubmit fix.
